utils/validation/waitlist.ts

```diff
@@ -0,0 +1,20 @@
+import { WaitlistInput } from "models/Waitlist";
+
+/**
+ * validateWaitlistRecord takes input of type WaitlistInput and validates the input
+ * @param input - data corresponding to a waitlist
+ * @returns string[] - array of errors of type string
+ */
+export function validateWaitlistRecord(input: WaitlistInput): string[] {
+    const validationErrors: string[] = [];
+
+    if (!input.classId) {
+        validationErrors.push("classId is not provided");
+    }
+
+    if (!input.parentId) {
+        validationErrors.push("parentId is not provided");
+    }
+
+    return validationErrors;
```

Comment:
Do we need to check for duplicates, like what if the class id and parent id are already in the waitlist records?
Or what if the ids are not valid (like if they are not ints, or dont exist)? 
Sorry if this is already checked somewhere else😅

Reply:
I believe if the class id and parent id are already in the waitlist record, since the pair is the primary key of the table, prisma should throw an error. We could handle it before it reaches prisma but that would mean an additional query. Do you think we should change it to that? @cindy-wang328

Reply:
@cindy-wang328 I'll merge this since it's not blocked on it but if you think we should do this, we can make that change in the future. If we do make the change, do you think we should check for existence of a user, a program, etc as well prior to creating?